src/diffusers/modeling_utils.py

```diff
@@ -615,10 +610,6 @@ def _get_model_file(
                     subfolder=subfolder,
                     revision=revision,
                 )
-                send_telemetry(
```

Comment:
This we can and should leave IMO because it's when people download from the Hub so IP is exposed anyways

Reply:
The pipeline class is already sent via the hub request, so this call was redundant.

Reply:
Ok all good then!